py/aou_workbench_client/cohorts.py

```diff
@@ -1,48 +1,40 @@
 from aou_workbench_client.auth import get_authenticated_swagger_client
 from aou_workbench_client.config import all_of_us_config
 from aou_workbench_client.swagger_client.apis.cohorts_api import CohortsApi
-from copy import deepcopy
 
-def materialize_cohort_page(request, debug=False):
-    """Returns a MaterializeCohortResponse representing a page of results
-    from materializing a cohort in the workspace containing this notebook, based
-    on the provided MateralizeCohortRequest."""
+def get_data_table_query(data_table_specification, debug=False):
+    """Returns a CdrQuery that can be used to retrieve data from BigQuery for the
+    specified data table specification.
+
+    :param data_table_specification: the DataTableSpecification to get the query for
+    :param debug: true if debug request and response information should be
+    displayed; defaults to false.
+    :return a query with the SQL and configuration to use when retrieving data
```

Comment:
Should document the type that it returns

Reply:
Done.